cmd/cloud-api-adaptor/main.go

```diff
@@ -71,6 +71,7 @@ func (cfg *daemonConfig) Setup() (cmd.Starter, error) {
 			flags.StringVar(&tlsConfig.KeyFile, "cert-key", "", "cert key")
 			flags.BoolVar(&tlsConfig.SkipVerify, "tls-skip-verify", false, "Skip TLS certificate verification - use it only for testing")
 			flags.BoolVar(&disableTLS, "disable-tls", false, "Disable TLS encryption - use it only for testing")
+			flags.IntVar(&cfg.serverConfig.ProxyTimeout, "proxy-timeout", 3, "Maximum timeout in minutes for establishing agent proxy connection")
```

Comment:
Would it make sense to have a variable in entrypoint.sh for modifying it (similarly to pause image: https://github.com/confidential-containers/cloud-api-adaptor/blob/e12e76fd6cce33524c52cb93365634a862c63632/entrypoint.sh#L13)

Reply:
I agree.

@Amulyam24 could you also update `entrypoint.sh` as well?

Reply:
Sure @yoheiueda. Made the changes, PTAL.